### PR TITLE
br: add restore sql binding steps

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-manual.md [hunk 1]
```diff
@@ -215,6 +219,39 @@ br restore full \
...
+    --log-file restore_system_table.log
+```
+
+恢复完成后，可以通过 [`SHOW GLOBAL BINDINGS`](../sql-statements/sql-statement-show-bindings.md) 检查执行计划绑定信息的恢复情况。
+
+```sql
+mysql> SHOW GLOBAL BINDINGS;
+```
+
+当前执行计划绑定信息在备份恢复后的动态加载仍在优化中 [#46527](https://github.com/pingcap/tidb/issues/46527), [#46528](https://github.com/pingcap/tidb/issues/46528)，需要手动刷新执行计划绑定信息。
```
Comment (songrijie) [inline br/br-snapshot-manual.md:244]: "ADMIN RELOAD BINDINGS" 是不是可以先作为流程的一步，没有必要提 46528，这个可以后续作为一个增强。

File: br/br-snapshot-manual.md [hunk 2]
```diff
@@ -7,16 +7,20 @@ summary: 介绍备份与恢复 TiDB 集群快照的命令行。
...
+    - [备份单张表的数据](#备份单张表的数据)
+    - [使用表库过滤功能备份多张表的数据](#使用表库过滤功能备份多张表的数据)
+  - [备份数据加密](#备份数据加密)
+  - [恢复快照备份数据](#恢复快照备份数据)
+  - [恢复备份数据中指定库表的数据](#恢复备份数据中指定库表的数据)
+    - [恢复单个数据库的数据](#恢复单个数据库的数据)
+    - [恢复单张表的数据](#恢复单张表的数据)
+    - [使用表库功能过滤恢复数据](#使用表库功能过滤恢复数据)
+    - [恢复系统表中存储的执行计划绑定信息](#恢复系统表中存储的执行计划绑定信息)
+  - [恢复加密的快照备份数据](#恢复加密的快照备份数据)
```
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:23]:
```suggestion
- [备份集群快照](#备份集群快照)
- [备份 TiDB 集群指定库表的数据](#备份-tidb-集群指定库表的数据)
    - [备份单个数据库的数据](#备份单个数据库的数据)
    - [备份单张表的数据](#备份单张表的数据)
    - [使用表库过滤功能备份多张表的数据](#使用表库过滤功能备份多张表的数据)
- [备份数据加密](#备份数据加密)
- [恢复快照备份数据](#恢复快照备份数据)
- [恢复备份数据中指定库表的数据](#恢复备份数据中指定库表的数据)
    - [恢复单个数据库的数据](#恢复单个数据库的数据)
    - [恢复单张表的数据](#恢复单张表的数据)
    - [使用表库功能过滤恢复数据](#使用表库功能过滤恢复数据)
    - [恢复系统表中存储的执行计划绑定信息](#恢复系统表中存储的执行计划绑定信息)
- [恢复加密的快照备份数据](#恢复加密的快照备份数据)
```
“TiDB 快照备份与恢复命令行手册”是一级标题，不需要在这里列出

File: br/br-snapshot-manual.md [hunk 3]
```diff
@@ -215,6 +219,39 @@ br restore full \
     --log-file restorefull.log
 ```
 
+### 恢复系统表中存储的执行计划绑定信息
+
+如果你需要恢复原集群的执行计划绑定信息，执行 `br restore full` 命令，并用 `--with-sys-table` 指定说明需要恢复系统表，通过 `--filter` 或 `-f` 指定恢复 `mysql.bind_info` 表。
```
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:224]:
```suggestion
如果你需要恢复原集群的执行计划绑定信息，执行 `br restore full` 命令，设置 `--with-sys-table` 并通过 `--filter` 或 `-f` 指定需要恢复的系统表。
```

File: br/br-snapshot-manual.md [hunk 4]
```diff
@@ -215,6 +219,39 @@ br restore full \
...
+```shell
+br restore full \
+    --pd "${PD_IP}:2379" \
+    --filter 'mysql.bind_info' \
+    --with-sys-table \
+    --ratelimit 128 \
+    --storage "s3://${backup_collection_addr}/snapshot-${date}?access-key=${access-key}&secret-access-key=${secret-access-key}" \
+    --log-file restore_system_table.log
+```
+
```
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:237]:
```suggestion

关于恢复系统表的更多信息，请参考[恢复 `mysql` 数据库下的表](/br/br-snapshot-guide.md#恢复-mysql-数据库下的表)。

```

File: br/br-snapshot-manual.md [hunk 5]
```diff
@@ -215,6 +219,39 @@ br restore full \
...
+恢复完成后，可以通过 [`SHOW GLOBAL BINDINGS`](../sql-statements/sql-statement-show-bindings.md) 检查执行计划绑定信息的恢复情况。
```
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:238]:
```suggestion
恢复完成后，可以通过 [`SHOW GLOBAL BINDINGS`](/sql-statements/sql-statement-show-bindings.md) 检查执行计划绑定信息的恢复情况：
```

File: br/br-snapshot-manual.md [hunk 6]
```diff
@@ -215,6 +219,39 @@ br restore full \
...
+
+```sql
+mysql> SHOW GLOBAL BINDINGS;
```
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:241]:
```suggestion
SHOW GLOBAL BINDINGS;
```
为了方便用户直接在文档站中复制代码块，不建议添加 mysql>

File: br/br-snapshot-manual.md (hunk 1 above)
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:244]:
```suggestion
当前执行计划绑定信息在备份恢复后的动态加载仍在优化中（相关的 issue 为 [#46527](https://github.com/pingcap/tidb/issues/46527) 和 [#46528](https://github.com/pingcap/tidb/issues/46528)，你需要手动刷新执行计划绑定信息。
```

File: br/br-snapshot-manual.md [hunk 7]
```diff
@@ -215,6 +219,39 @@ br restore full \
...
+
+当前执行计划绑定信息在备份恢复后的动态加载仍在优化中 [#46527](https://github.com/pingcap/tidb/issues/46527), [#46528](https://github.com/pingcap/tidb/issues/46528)，需要手动刷新执行计划绑定信息。
+```sql
+-- 确保 mysql.bind_info 表中 builtin_pseudo_sql_for_bind_lock 的记录仅 1 行，如果多于 1 行，需要手动删除
+mysql> select count(*) from mysql.bind_info where original_sql = 'builtin_pseudo_sql_for_bind_lock';
+
+mysql> delete from bind_info where original_sql = 'builtin_pseudo_sql_for_bind_lock' limit 1;
+
+-- 强制重新加载 Binding 信息
+mysql> ADMIN RELOAD BINDINGS;
```
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:252]:
```suggestion
-- 确保 mysql.bind_info 表中 builtin_pseudo_sql_for_bind_lock 的记录仅 1 行，如果多于 1 行，需要手动删除
SELECT count(*) FROM mysql.bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock';
DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LIMIT 1;

-- 强制重新加载绑定信息
ADMIN RELOAD BINDINGS;
```

File: br/br-snapshot-manual.md [hunk 8]
```diff
@@ -215,6 +219,39 @@ br restore full \
...
+    --log-file restore_system_table.log
+```
+
+恢复完成后，可以通过 [`SHOW GLOBAL BINDINGS`](../sql-statements/sql-statement-show-bindings.md) 检查执行计划绑定信息的恢复情况。
+
+```sql
+SHOW GLOBAL BINDINGS;
+```
+
+当前执行计划绑定信息在备份恢复后的动态加载仍在优化中（相关的 issue 为 [#46527](https://github.com/pingcap/tidb/issues/46527) 和 [#46528](https://github.com/pingcap/tidb/issues/46528)，你需要手动刷新执行计划绑定信息。
```
Comment (Oreoxmt) [inline br/br-snapshot-manual.md:244]:
```suggestion
当前执行计划绑定信息在备份恢复后的动态加载仍在优化中（相关的 issue 为 [#46527](https://github.com/pingcap/tidb/issues/46527) 和 [#46528](https://github.com/pingcap/tidb/issues/46528)），你需要手动刷新执行计划绑定信息。
```

File: br/br-snapshot-manual.md (hunk 4 above)
Comment (benmaoer) [reply br/br-snapshot-manual.md:236]: 不建议在这里链接到 [恢复 `mysql` 数据库下的表]，链接部分的文档还未经过 QA 验证

**Comment by Oreoxmt** [comment]
@songrijie @BornChanger PTAL again

File: br/br-snapshot-manual.md [hunk 9]
```diff
@@ -215,6 +218,39 @@ br restore full \
...
+如果你需要恢复原集群的执行计划绑定信息，执行 `br restore full` 命令，设置 `--with-sys-table` 并通过 `--filter` 或 `-f` 指定需要恢复的系统表。
+
+下面是恢复 `mysql.bind_info` 表的示例：
+
+```shell
+br restore full \
+    --pd "${PD_IP}:2379" \
+    --filter 'mysql.bind_info' \
+    --with-sys-table \
+    --ratelimit 128 \
```
Comment (3pointer) [inline br/br-snapshot-manual.md:232]: restore 的时候，感觉默认可以不用设置 ratelimit

File: br/br-snapshot-manual.md [hunk 10]
```diff
@@ -215,6 +218,39 @@ br restore full \
...
+    --log-file restore_system_table.log
+```
+
+恢复完成后，可以通过 [`SHOW GLOBAL BINDINGS`](/sql-statements/sql-statement-show-bindings.md) 检查执行计划绑定信息的恢复情况：
+
+```sql
+SHOW GLOBAL BINDINGS;
+```
+
+当前执行计划绑定信息在备份恢复后的动态加载仍在优化中（相关的 issue 为 [#46527](https://github.com/pingcap/tidb/issues/46527) 和 [#46528](https://github.com/pingcap/tidb/issues/46528)），你需要手动刷新执行计划绑定信息。
```
Comment (BornChanger) [inline br/br-snapshot-manual.md:243]: 要不要等这2个 issue 关了, 再合并呢?

Comment (benmaoer) [reply br/br-snapshot-manual.md:242]: 这两个 issue 还没有确定排期修复，估计要等很长时间

**Review by BornChanger — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**